### PR TITLE
Test change

### PR DESCRIPTION
**Pull request by RogerJCarlson**

### CODE REVIEW

File: models/staging/synthea/_synthea__models.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (katy-sadowski) [inline models/staging/synthea/_synthea__models.yml:1]: hm, not sure about this

**Review by katy-sadowski — CHANGES_REQUESTED** [review]
we should not do this.

**Comment by RogerJCarlson** [comment]
F

**Comment by RogerJCarlson** [comment]
Fine!

**Pull request closed without merging**